client/miner.c

```diff
@@ -421,7 +421,7 @@ int xdag_send_block_via_pool(struct xdag_block *b)
 /* picks random pool from the list of pools */
 int xdag_pick_pool(char *pool_address)
 {
-	char addresses[30][50] = {0};
+	char addresses[30][50] = {{0}, {0}};
```

Comment:
Any issue met?

Reply:
/git/xdag/client/miner.c:424:28: warning: suggest braces around initialization of subobject [-Wmissing-braces]
        char addresses[30][50] = {0};
fix compile warning